public/js/business_page.js

```diff
@@ -0,0 +1,18 @@
+const getElement = id => document.getElementById(id);
```

Comment:
document is undefined and will get you a warning from eslint.
use ```/* global document */``` at the top of the file to avoid that.